js/archive.js

```diff
@@ -6,28 +6,45 @@ layout: null
 if (window.navigator.onLine) {
   var suppressButterBar = false;
   /* This JSON file contains a current list of all docs versions of Docker */
-  $.getJSON("/js/archives.json", function(result){
-    var outerDivStart = '<div id="archive-butterbar"><div class="container"><div style="text-align: center"><span id="archive-list">This is <b><a href="https://docs.docker.com/docsarchive/" style="color: #254356; text-decoration: underline !important">archived documentation</a></b> for Docker&nbsp;' + dockerVersion + '. Go to the <a style="color: #254356; text-decoration: underline !important" href="https://docs.docker.com/">latest docs</a> or a different version:&nbsp;&nbsp;</span>' +
-                               '<span style="z-index: 1001" class="dropdown">';
-    var listStart = '<ul class="dropdown-menu" role="menu" aria-labelledby="archive-menu">';
-    var listEnd = '</ul>';
-    var outerDivEnd = '</span></div></div></div>';
+  $.getJSON("/js/archives.json", function (result) {
+    var outerDivStart =
+      '<div id="archive-butterbar"><div class="container"><div style="text-align: center"><span id="archive-list">This is <b><a href="https://docs.docker.com/docsarchive/" style="color: #254356; text-decoration: underline !important">archived documentation</a></b> for Docker&nbsp;' +
+      dockerVersion +
+      '. Go to the <a style="color: #254356; text-decoration: underline !important" href="https://docs.docker.com/">latest docs</a> or a different version:&nbsp;&nbsp;</span>' +
```

Comment:
So, if we're not gonna be publishing the current (19.03) version of the docs as an archive, do we need this script at all? Basically this script would only be actively running on archived versions

Reply:
Yup, that's correct. As per the earlier discussions, the current version will not be archived and there will be only one version of the docs on docs.docker.com